Changelogs for Digital Twin Manager 1020.1.11 (#2217)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/dtm-1020.1.11-update-label-capitalization.md

---
date: ""

> title: "Update label capitalization"

Reviewer: ```suggestion
title: "Updated label capitalization"
```

product_area: "Application enablement & solutions"
change_type:
    - value: "change-VSkj2iV9m"
      label: "Fix"
component:
    - value: "component-Tl88RYb4A"
      label: "Digital Twin Manager"
build_artifact:
    - value: "tc-wYIY0MBDO"
      label: "dtm"
ticket: "CTM-1397"
version: "1020.1.11"
---

> Fixed inconsistent capitalization of labels, error messages and info messages to a uniform style throughout DTM.

Reviewer: ```suggestion
Inconsistent capitalization of labels, error messages and info messages has been unified throughout the Digital Twin Manager application.
```

> Inconsistent capitalization of labels, error messages and info messages has been unified throughout the Digital Twin Manager application.

Reviewer 2: ```suggestion
Inconsistent capitalization of labels, error messages and info messages has been unified throughout the Digital twin manager application.
```
